daemon/launch_unix.go

```diff
@@ -244,6 +248,19 @@ func LaunchDaemons(ctx context.Context, launchers []Launcher, egrp *errgroup.Gro
 					metrics.SetComponentHealthStatus(metrics.HealthStatusComponent(metricName), metrics.StatusCritical,
 						launchers[chosen].Name()+" process failed unexpectedly")
 					err = errors.Wrapf(waitResult, "%s process failed unexpectedly", launchers[chosen].Name())
+					service_key := "origin"
+					if config.IsServerEnabled(server_structs.CacheType) {
+						service_key = "cache"
+					}
+					crashTimestamp := time.Now().Unix()
+					log.Debug("Recording xrootd crash at time: ", crashTimestamp)
+
+					dbErr := database.CreateOrUpdateCounter(service_key, int(crashTimestamp))
+					if dbErr != nil {
+						log.Debug("Error recording xrootd crash: ", dbErr)
+						return errors.Wrap(err, "Unable to record xrootd crash")
```

Comment:
Is there any benefit in also wrapping/returning the database error? Also, since `dbErr` isn't used later, can you reformat to:
```
if dbErr := yourFunc(stuff); dbErr != nil {
    // your stuff
}
```